notebook/notebook/notebook.py

```diff
@@ -389,9 +390,10 @@ def notebook_status_from_pod(pod):
             new_status = await notebook_status_from_notebook(k8s, service, headers, cookies, notebook)
             changed = await update_notebook_return_changed(dbpool, user_id, notebook, new_status)
             if changed:
+                log.info(f"pod {notebook['pod_name']} status changed: {notebook['state']} => {new_status['state']}")
                 break
         except Exception:  # pylint: disable=broad-except
-            log.exception('while updating status in /wait')
+            log.exception(f"/wait: error while checking or updating status for pod: {notebook['pod_name']}")
```

Comment:
Sorry to be pedantic here, but I don't understand what this message means anymore.  What's checking vs updating?  We're not using the term check anywhere else in this code.  If you check on the status of the pod, your understanding is still updated (that is, it is more up to date) even if the status hasn't changed.

Reply:
Sure. Checking was a reference to "notebook_status_from_notebook" which fetched the current status of the pod. 

I reverted it, let me know if you'd like something else.